src/pages/CropListPage.js

```diff
@@ -1,22 +1,43 @@
+import styled from 'styled-components/macro'
 import CropItem from '../components/CropItem'
 import PropTypes from 'prop-types'
+import Button from '../components/Button'
 
 CropListPage.propTypes = {
   crops: PropTypes.array.isRequired,
-  onNavigate: PropTypes.func.isRequired,
+  onClickDetails: PropTypes.func.isRequired,
+  onClickFavorites: PropTypes.func.isRequired,
 }
 
-export default function CropListPage({ crops, onNavigate }) {
+export default function CropListPage({
+  crops,
+  onClickDetails,
+  onClickFavorites,
+}) {
   return (
-    <div>
-      {crops.map(({ id, attributes }) => (
-        <CropItem
-          key={id}
-          name={attributes.name}
-          image={attributes.main_image_path}
-          onClick={() => onNavigate(id)}
-        />
-      ))}
-    </div>
+    <Wrapper>
+      <Container>
+        {crops.map(({ id, attributes }) => (
+          <CropItem
+            key={id}
+            name={attributes.name}
+            image={attributes.main_image_path}
+            onClick={() => onClickDetails(id)}
+          />
+        ))}
+      </Container>
+
+      <Button onClick={onClickFavorites}>My Garden</Button>
+    </Wrapper>
   )
 }
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-rows: auto 48px;
+  height: 100vh;
+`
+
+const Container = styled.div`
+  overflow-y: scroll;
```

Comment:
if you set it to auto instead of scroll, the scrollbar is only visible when needed (I just learned this so I am not quite sure how it really works, but I thought that it is quite neat)